bokeh/sampledata/stocks.py

```diff
@@ -67,20 +68,29 @@
 # Private API
 #-----------------------------------------------------------------------------
 
-def _read_data(name):
+class StocksData(TypedDict):
```

Comment:
This structure holds info for a single stock so `StockData` (or maybe `TickerData`) would be better I think